src/Commands/Release/Release.php

```diff
@@ -258,12 +258,20 @@ protected function getPublishCommand($version, $project)
     }
 
     /**
-     * Whether to include all commits in the changelog
+     * Whether to include all commits in the changelog. Will look at project level configuration as well as the
+     * specific argument passed to the command.
      *
      * @return bool
      */
     public function getIncludeOtherChanges()
     {
-        return (bool) $this->input->getOption('include-other-changes');
+        // If project level configuration is defined, use that
+        $projectConfig = $this->getProject()->getChangelogIncludeOtherChanges();
+        if ($projectConfig !== null) {
+            return $projectConfig;
```

Comment:
Do you want to fall back to the option if this is explicitly set `false`?

Reply:
It's tricky to handle arguments because they never come out as null - always a boolean.

I guess we could make the logic:

* if project config is null, return argument value
* if project config is true, return true
* if project config is false and argument value is true, return true
* if project config is false and argument value is false, return false

It'd just require a bit of nesting. Will update.

Reply:
There isn't really a way to explicitly set the argument to false. We could change the argument type to one that accepts values instead of just being a "it's there or it isn't" argument.

Reply:
There's a naughty way we can workaround this by using `getParameterOption()` in combination with `getOption()` since it returns false when the option was not explicitly passed. Please hold.

Reply:
For now it's easiest if you change the project config to set it to false if you don't want it. It'll get messy otherwise and I don't want to add a second negative argument just for this